apps/nextjs-example/pages/index.tsx

```diff
@@ -340,6 +371,95 @@ function OptionalFunctionality() {
         );
     };
 
+    const onSubmitScript = async () => {
+        const payload = {
+            type: "script_payload",
+            code: {
+                bytecode: DO_NOTHING_SCRIPT
+            },
+            type_arguments: [],
+            arguments: []
+        };
+
+        const response = await signAndSubmitTransaction(payload);
+        try {
+            if (response?.hash === undefined) {
+                throw new Error(`No response given ${response}`)
+            }
+            await aptosClient(network?.name.toLowerCase()).waitForTransaction(response.hash);
+            setSuccessAlertHash(response.hash, network?.name);
+        } catch (error) {
+            console.error(error);
+        }
+        setSuccessAlertMessage(
+            JSON.stringify({signAndSubmitTransaction: response})
+        );
+    };
+
+    // TODO: Let's put this into a cookie or local storage so it only happens once
+    const createFeePayerAccount = async (client: AptosClient) => {
+        // Already exists, so lets not fund it
+        if (feePayerAccount) {
+            return feePayerAccount;
+        }
+
+        let newAccount = new AptosAccount();
+        let feePayerAddress = newAccount.address();
+
+        try {
+            let coinClient = new CoinClient(client);
+            let balance = await coinClient.checkBalance(feePayerAddress);
+            if (Number(balance) > 10000) {
+                return newAccount;
+            }
+        } catch (error: any) {
+            // Swallow errors, likely due to account not existing
+        }
+        await (faucet(network?.name.toLowerCase()).fundAccount(feePayerAddress, 100000));
+
+        return newAccount;
+    }
+
+    const onSubmitFeePayer = async () => {
+        if (!account) {
+            throw new Error("Not connected");
+        }
+        let provider = aptosClient(network?.name.toLowerCase());
+        // Generate an account and fund it
+        let feePayerAccount = await createFeePayerAccount(provider);
+
+        const payload: Types.TransactionPayload = {
+            type: "entry_function_payload",
+            function: "0x1::coin::transfer",
```

Comment:
nit
```suggestion
            function: "0x1::aptos_account::transfer",
```
then dont need the `type_arguments`